nav2_bringup/params/nav2_params.yaml

```diff
@@ -26,7 +26,7 @@ amcl:
     recovery_alpha_fast: 0.0
     recovery_alpha_slow: 0.0
     resample_interval: 1
-    robot_model_type: "differential"
+    robot_model_type: "nav2_amcl::DifferentialMotionModel"
```

Comment:
There are other yaml files in this directory that would need to be updated as well (and in the `nav2_system_tests`, try searching for `robot_model_type` and replacing with this)

Reply:
ohhhh that actually might be why your tests are failing!

Reply:
Changed the robot_model_type in the yaml files wherever required.